db/models.py

```diff
@@ -50,3 +61,46 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(default=now)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+
+    def __str__(self) -> str:
+        return f"{self.created_at.strftime("%Y-%m-%d %H:%M:%S")}"
```

Comment:
The `__str__` method in the `Order` model uses double quotes inside a string that is also enclosed in double quotes. This will cause a syntax error. Consider using single quotes for the outer string or escaping the inner double quotes.